composer.json

```diff
@@ -39,6 +42,12 @@
   "extra": {
     "flarum-extension": {
       "title": "FoF Best Answer",
+      "category": "feature",
+      "info": {
```

Comment:
We actually redid this piece recently, to use more of the native composer fields: https://github.com/flarum/core/blob/master/src/Extension/Extension.php#L352

For an example of the new system, see pretty much any bundled extension, I updated all of them today.

Reply:
Perfect, I'll update...